src/components/account/account-dropdown.vue

```diff
@@ -0,0 +1,82 @@
+<!--
+  - Copyright (c) TIKI Inc.
+  - MIT license. See LICENSE file in root directory.
+  -->
+
+<script setup lang="ts">
+import { ref, watch } from "vue";
+import { icon } from "@/service/receipt/receipt-account-type";
+const emits = defineEmits(["input"]);
+const props = defineProps({
+  accounts: {
```

Comment:
Why is the entire account passed in? This should simply pass back and forth the selected value (either a string or enum) and it should be set up to use a v-model